postgresql-ballerina/tests/resources/sql-scripts/connection-pool/connection-pool1-test.sql

```diff
@@ -0,0 +1,16 @@
+        DROP TABLE IF EXISTS Customers;
```

Comment:
Reformat

Reply:
I will update

Reply:
Updated. Commit id is 6b36c0ef014b8278f8bffcab8f51e02fced50da2